### PR TITLE
Topic detail with consumer offsets

### PR DESCRIPTION
**Pull request by durdina**
Hi,
we extended the existing topic detail page. We added a "More details..." button that expands the topic detail page and adds a table with current offsets for all consumers on a given topic. Now the topic detail page not only displays consumer names but it also displays current offsets for those consumers on a single page. 

This extension proved to be useful for us while we were migrating consumers from old topic to a new topic and therefore we wanted to monitor old topic until we saw no lag on any of those consumers. On a  new topic we wanted to monitor that all consumers now switched to it are just fine and happily consuming messages. Although the same could be achieved in original state it was crucial for us to see those details on a single page at the same time.

Technically we extracted respective part of group.html into group.html.inc and included that part into both group.html and a new page topic-consumers.html which is now opened from topic-detail.html. We also splitted consumers on a page accoring to their activity so in the end we display two tables - one for active consumers and one for inactive consumers.

Screenshot of a changed page is here:
[Screenshot 1 - Topic detail page with a new button More details...](https://www.evernote.com/shard/s131/sh/3512ffd3-8bc9-463c-9ed6-aaa87a1e7f3e/4d4ff5bff237ad313a332fc71c6108df)
[Screenshot 2 - The page reloaded after clicking More details ...](https://www.evernote.com/shard/s131/sh/397fba76-f256-43b5-9ff5-c421bbc9b6ed/05e36d0f755e5b44e789739ea516b97f)

### CODE REVIEW

**Comment by Mortimerp9** [comment]
This looks awesome, I like some of the refactor too. Can one of you sent us a copy of the contributor agreement linked here:
https://github.com/quantifind/KafkaOffsetMonitor#contributing

sadly, I am not allowed to merge without that.

File: project/Build.scala [hunk 1]
```diff
@@ -5,8 +5,38 @@ import AssemblyKeys._
...
+      val ivyCredentials = Path.userHome / ".ivy2" / ".credentials"
+      val mavenCredentials = Path.userHome / ".m2" / "settings.xml"
+      (ivyCredentials.asFile, mavenCredentials.asFile) match {
+        case (ivy, _) if ivy.canRead => Credentials(ivy) :: Nil
+        case (_, mvn) if mvn.canRead => loadMavenCredentials(mvn)
+        case _ => Nil
+      }
+    }
+  )
+
```
Comment (Mortimerp9) [inline project/Build.scala]: this seems very specific to your company publish repo

Comment (durdina) [inline project/Build.scala]: My apologies. Please ignore this single change on merging. 
I am not really sure how else could this single change be removed given that pull request is already existing.

**Comment by durdina** [comment]
Thank you for preaccepting the request. I will try to process the necessary contributor agreement within our company as soon as possible.

**Comment by durdina** [comment]
Hi there, 
Please find the two licence agreements signed by us as contributors in contributors@quantifind.com.

Thank you for your patience.

**Comment by Mortimerp9** [comment]
excellent, the code looks good to me, I want to test it and then I'll merge the PR next week.

**Pull request merged**